src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -0,0 +1,351 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0" DefaultTargets="Build">
+   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.legacy.props" />
+   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.common.props')" />
+   <Import Project="$(VSSDKRoot)\build\Microsoft.VSSDK.BuildTools.props" />
+   <PropertyGroup>
+      <Shipping>true</Shipping>
+      <ResolveNuGetPackages>true</ResolveNuGetPackages>
+      <PackagesDirectory>$(UserProfile)\.nuget\packages</PackagesDirectory>
+      <TargetFrameworkProfile />
```

Comment:
Discussed offline with @emgarten, I thought these were added during the migration, but it seems like they're redundant things. 
Will create an issue that deals with the cleanup of these redundant props